modules/nf-core/ctatsplicing/prepgenomelib/main.nf

```diff
@@ -0,0 +1,125 @@
+process CTATSPLICING_PREPGENOMELIB {
+    tag "$meta.id"
+    label 'process_single'
+    stageInMode 'copy'
+
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://data.broadinstitute.org/Trinity/CTAT_SINGULARITY/CTAT-SPLICING/ctat_splicing.v0.0.2.simg' :
+        'docker.io/trinityctat/ctat_splicing:0.0.2' }"
```

Comment:
The main reason I couldn't add these modules to the nf-core/modules repository is because of the containers. These are third party containers so we don't really have any control over what happens with these... I think it would be worth it to either look into adding these modules to bioconda first or at least add these containers to the nf-core quay repo if bioconda isn't an option

Reply:
i'm guessing bioconda is an option because i see other ctat packages have recipes there. i am willing to give it a shot.

Reply:
@nvnieuwk i'm just getting back to this. it sounds like there's already been a considerable effort to put this tool into bioconda: https://github.com/TrinityCTAT/CTAT-SPLICING/issues/12 and the developer has not made any significant effort to simplify the dependencies. would it be possible to approve these anyways as sort of the "best-effort", since it currently works and has been working for ~3 years? If not, is there another place where we can store a redundant copy of these containers that we feel more comfortable using? 

if not i will make some time in the next week or so to try the bioconda. just want to make sure that this is really necessary given what seems like unlikely success.

Reply:
Hi, we have a couple of modules where this is also the case. We just need to push this container to nf-core docker registry to ensure full stability of the container location. I can do this some time this week! (please remind me if I haven't done it by next week)

Reply:
Yes, I also think uploading this to nf-core docker registry would be the way to go! Thanks @anoronh4 :)

Reply:
i didn't know there was something like that available! awesome, thank you!